### PR TITLE
Remove maven central repository, use only jcenter

### PR DESCRIPTION
**Pull request by yinonavraham**
jcenter is a super set on top of maven central, so having both of those repositories is redundant, and the preferred one should be jcenter.

CheckList:

  * [x] Update API docs for the component.
  * [x] Update/Add demo to demonstrate new feature.
  * [x] Update test scripts for the feature.
  * [x] Add unit tests for the feature.

(checked, although not really relevant - not a feature nor bug...)

### CODE REVIEW

**Comment by misakuo** [comment]
Thanks for your contribution.  we can remove the `mavenCentral` from repositories, but I think to replace it with `mavenLocal` will be better?

**Comment by yinonavraham** [comment]
Certainly not. `mavenLocal` is your local maven cache (i.e. the `.m2` directory). Doing so will remove any central maven repository. You should at least have either `jcenter` or `mavenCentral`, I defenetly prefer `jcenter`.

**Comment by misakuo** [comment]
I mean 
```groovy
 repositories {
        mavenLocal()
        jcenter()
  }
```
First find in the local maven cache, if not found, find in jcenter

**Comment by yinonavraham** [comment]
Ah, sure, that's ok, sorry for the misunderstanding. I think it is implicit, but it's ok to have it anyway.

File: android/build.gradle [hunk 1]
```diff
@@ -1,7 +1,6 @@
 
 buildscript {
     repositories {
-        mavenCentral()
```
Comment (misakuo) [inline android/build.gradle]: Replace it with `mavenLocal`

**Pull request closed without merging**